File: DellProjectAPI/DellProjectAPI/Models/Event.cs
using System;
using System.Collections.Generic;

namespace DellProjectAPI.Models
{
    public class Event
    {
        public Event()
        {
        }

        public DateTime Date { get; set; }
        public string EventName { get; set; }
        public int Id { get; set; }
        public List<Attendee> Attendees { get; set; }
        public int Duration { get; set; }

        public Event(Event that)
        {
            Date = that.Date;
            EventName = that.EventName;
            Id = that.Id;
            Attendees = that.Attendees;
            Duration = that.Duration;
        }

        public Event(int id, string name, DateTime date, List<Attendee> attendees, int duration)
        {
            Id = id;
            EventName = name;
            Date = date;
            Duration = duration;
            Attendees = attendees;
        }

    }
}

File: PlannerFrontEnd/src/app/interfaces/Event.ts
import { Activity } from "./Activity";
import { Attendee } from "./Attendee";

export interface Event{
    id? : number,
    activities?: Activity[],
    organizerId? : number,
    date : Date,
    eventName : string,
    duration? : number,
    attendees? : Attendee[],
    category : string,
    time: string,
    location: string

}